src/app/[locale]/topics/[topicId]/(topic)/components/tabs/MindMapTab.tsx

```
@@ -8,8 +8,19 @@ import DataStatus from '@/components/errors/DataStatus';
 
-export default function MindmapTab() {
+interface StudentProps {
+    role: UserRoleEnum.USER;
+}
+interface TeacherProps {
+    role?: UserRoleEnum.TEACHER;
+}
+
+type Props = StudentProps | TeacherProps;
+
+export default function MindmapTab({}: Props) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)